test/k8up.mk

```
@@ -0,0 +1,27 @@
+k8up_sentinel = $(kind_dir)/k8up_sentinel
+minio_sentinel = $(kind_dir)/minio_sentinel
+
+.PHONY: k8up-setup
+k8up-setup: minio-setup $(k8up_sentinel) ## Install K8up operator
+
+$(k8up_sentinel): export KUBECONFIG = $(KIND_KUBECONFIG)
+$(k8up_sentinel): $(KIND_KUBECONFIG)
+	helm repo add appuio https://charts.appuio.ch
```

Throws an error if the repo already exists.

```suggestion
	helm repo add appuio https://charts.appuio.ch || true
```
or
```suggestion
	helm repo add --force-update appuio https://charts.appuio.ch
```